web/client/src/models/initial.ts

```diff
@@ -8,19 +8,25 @@ export class ModelInitial<T extends object = any> {
 
   isModel = true
 
-  constructor(initial: Initial<T> | InitialWithId<T>) {
-    this._initial = isNotNil(initial?.id)
-      ? (initial as InitialWithId<T>)
-      : new Proxy<InitialWithId<T>>(
-          Object.assign(initial ?? {}, {
-            id: uid(),
-          }),
-          {
-            set() {
-              throw new Error('Cannot change initial file')
+  constructor(initial?: Initial<T> | InitialWithId<T>) {
+    if (isNil(initial)) {
+      this._initial = Object.assign(initial ?? {}, {
```

Comment:
Do you need the nullish coalescing operator if `initial` is already known to be null?